test/test.bats

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bats
+
+load helpers
+
+setup() {
```

Comment:
`setup` and `teardown` will be executed after each team, do we want that behavior or do we want `setup_file` and `teardown_file`?

Reply:
changed to teardown_file and removed setup for right now to debug copa's connection to buildkitd

Reply:
moved setup code to build.yaml and it works now and kept teardown_file() to stop buildkit after the tests. I'm guessing the setup function was messing with the access between copa and buildkitd?